index.html

```diff
@@ -4280,17 +4280,6 @@ <h2>API design rationale</h2>
         into consideration when designing a Scripting API for WoT.
       </p>
     </section>
-    <section> <h4>Fetching and validating a TD</h4>
```

Comment:
I think this section is still needed, but needs to be updated with `requestThingDescription` in mind.

Reply:
I tried to add an initial proposal in e4f90d488a58df1528b3c1f8579df47e65594534 that can hopefully serve as the basis for further discussions :)